test/e2e/specs/editor/various/switch-to-draft.spec.js

```diff
@@ -57,7 +57,9 @@ test.describe( 'Clicking "Switch to draft" on a published/scheduled post/page',
 							.getByRole( 'button', { name: 'Close Settings' } )
 							.click();
 					}
-					await page.getByRole( 'button', { name: 'Save' } ).click();
+					await page
+						.getByRole( 'button', { name: 'Save', exact: true } )
+						.click();
```

Comment:
Suggestion (non-blocking): We can also add scope to the locator. Example #62454.